src/components/delegate-tag/delegate-tag.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, useMemo } from 'react';
+import { Text } from 'react-native';
+
+import { useSelectedBakerSelector } from '../../store/baking/baking-selectors';
+import { TEZ_TOKEN_SLUG } from '../../token/data/tokens-metadata';
+import { TokenInterface } from '../../token/interfaces/token.interface';
+import { getTokenSlug } from '../../token/utils/token.utils';
+import { isDefined } from '../../utils/is-defined';
+import ABContainer from '../ab-container/ab-container';
+import { DelegateTagA } from './components/delegate-ab-components/delegate-tag-a/delegate-tag-a';
+import { DelegateTagB } from './components/delegate-ab-components/delegate-tag-b/delegate-tag-b';
+import { DelegateTagContainer } from './components/delegate-tag-container/delegate-tag-container';
+import { useDelegateTagStyles } from './delegate-tag.styles';
+
+interface Props {
+  token: TokenInterface;
+  apy?: number;
+}
+
+export const DelegateTag: FC<Props> = ({ apy, token }) => {
+  const styles = useDelegateTagStyles();
+  const [, isBakerSelected] = useSelectedBakerSelector();
+
+  const tokenSlug = useMemo(() => getTokenSlug(token), [token]);
```

Comment:
Do we really need to memoize the string which is returned by a pretty simple function?